src/Tickets/Commerce/Gateways/Stripe/Gateway.php

```diff
@@ -69,10 +69,20 @@ public function get_admin_notices() {
 				'content' => __( 'Disconnecting from Stripe failed. Please try again.', 'event-tickets' ),
 				'type'    => 'error',
 			],
+			[
+				'slug' => 'tc-stripe-currency-mismatch',
+				'type' => 'notice',
+				'dismiss' => true,
+			],
 			[
 				'slug'    => 'tc-stripe-country-denied',
 				'content' => __( 'Due to Regulatory Issues between Stripe and the country listed in your Stripe account, the free version of Event Tickets cannot accept connections from accounts in your country. Please use a Stripe account from a different country or purchase Event Tickets Plus to continue.', 'event-tickets' ),
 				'type'    => 'error',
+			],
+			[
+				'slug'    => 'tc-stripe-account-disconnected',
+				'content' => __( 'Your stripe account was disconnected from the Stripe dashboard. If you believe this is an error, you can re-connect in the Event Tickets > Payment page.', 'event-tickets' ),
```

Comment:
```suggestion
				'content' => __( 'Your stripe account was disconnected from the Stripe dashboard. If you believe this is an error, you can re-connect in the Payment tab of the Settings page.', 'event-tickets' ),
```

Reply:
Maybe we can add a link to their WP Events > Settings > Payments so that they can get there without following the route and just by clicking a link.